shell/packages/sandstorm-db/db.js

```diff
@@ -2250,6 +2250,59 @@ if (Meteor.isServer) {
   };
 }
 
+if (Meteor.isServer) {
+  SandstormDb.prototype.deleteIdentity = function (identityId) {
+    this.removeApiTokens({ identityId: identityId });
+    this.collections.contacts.remove({ identityId: identityId });
+    Meteor.users.remove({ _id: identityId });
```

Comment:
If I have a save()'d identity capability, after this I should expect it to simply fail to restore()?